src/libraries/System.Text.RegularExpressions/gen/UpgradeToGeneratedRegexCodeFixer.cs

```diff
@@ -276,7 +276,10 @@ static RegexOptions GetRegexOptionsFromArgument(ImmutableArray<IArgumentOperatio
                 }
                 else
                 {
-                    return generator.LiteralExpression(argument.Value.ConstantValue.Value);
+                    SyntaxNode syntax = argument.Syntax;
+                    return syntax is ArgumentSyntax { NameColon: not null } argumentSyntax ?
+                        argumentSyntax.WithNameColon(null) :
+                        syntax;
```

Comment:
I haven't verified if this will work.

```suggestion
                    return argument.Value.Syntax;
```